app/views/course_user_data/sudo.html.erb

```diff
@@ -14,6 +14,6 @@
     <label for="student_autocomplete">Start typing student name or email</label>
   </div>
   <%= hidden_field_tag(:sudo_id) %>
-  <%= submit_tag "Sudo", {:value=>"Sudo", :class=> "btn primary"} %>
+  <%= submit_tag "Act as user", {:value=>"Act as user", :class=> "btn primary"} %>
```

Comment:
`:value=>...` seems to be redundant, so we can get rid of it.

Reply:
There also appears to be an unused sudo page under `/users/sudo.html.erb`, we can remove that as well.

Reply:
@najclark